### PR TITLE
REF: make arroots and maroots have consistent return type

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

**Comment by jbrockmendel** [comment]
@josef-pkt @ChadFulton this requires non-zero (though I hope non-massive) discussion.  What do ya'll think?

**Comment by jbrockmendel** [comment]
@josef-pkt thoughts on deprecation cycle?

File: statsmodels/tsa/tests/test_arima.py [hunk 1]
```diff
@@ -1987,6 +1987,18 @@ def test_predictions(self):
         predictions = self.arma_00_res.predict()
         assert_almost_equal(self.y.mean() * np.ones_like(predictions), predictions)
 
+    def test_arroots(self):
+        # regression test; older implementation of arroots returned None
+        # instead of en empty array
+        roots = self.arma_00_res.arroots
+        assert roots.size == 0
```
Comment (josef-pkt) [inline statsmodels/tsa/tests/test_arima.py]: numpy's assert_equal

**Comment by josef-pkt** [comment]
looks fine to me  (use numpy asserts for numbers)

summary uses explicit check for k_ar and k_ma, so will not be affected.

IMO, change is not important enough for FutureWarning or deprecation treatment, 
Deprecation cycle with option during transition would be difficult to add.

**Comment by jbrockmendel** [comment]
Just pushed, addressing comments.

Also updated lint.sh to keep directories together.

**Comment by josef-pkt** [comment]
all green, merging

@jbrockmendel thank you

**Pull request merged**